steal.lic

```diff
@@ -112,13 +111,13 @@ class Steal
 
   def do_buff(buffs, settings)
```

Comment:
Should this just be DRCA.do_buff?

Reply:
I think so; I hadn’t thought about it since I was focused on other things in this PR. I can test that

Reply:
I switched the buffing to DRCA.do_buffs